lib/KCard/index.vue

```diff
@@ -34,7 +35,7 @@
           as progressive loading experience.
         -->
         <span
-          v-if="$slots.thumbnailPlaceholder"
+          v-if="$slots.thumbnailPlaceholder && !thumbnailSrc"
```

Comment:
Here too we'd want to remove ` && !thumbnailSrc`.  Similar reason as other changes, see the comment on lines 34-35.